WinUIDesktopSample/App.xaml.cs

```diff
@@ -40,9 +41,18 @@ protected override void OnLaunched(LaunchActivatedEventArgs args)
             myWindow = window;
         }
 
-        private void Button_Click(object sender, RoutedEventArgs e)
+        private async void Button_Click(object sender, RoutedEventArgs e)
```

Comment:
just for testing? not intended for checkin?

Reply:
Probably a bad merge from when I initially started this work.